src/main/java/xizi/chatbot/Storage.java

```diff
@@ -0,0 +1,131 @@
+package xizi.chatbot;// Do not use wildcard imports
+
+import xizi.chatbot.task.Deadline;
+import xizi.chatbot.task.Event;
+import xizi.chatbot.task.Task;
+import xizi.chatbot.task.Todo;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.BufferedWriter;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+
+
+/**
+ * Handles the loading and saving of tasks from/to a file.
+ */
+public class Storage {
+    private final Path filePath;
+
+    /**
+     * Constructs a Storage object with the specified file path.
+     *
+     * @param filePath The path of the file where tasks are stored.
+     */
+    public Storage(String filePath) {
+        this.filePath = Paths.get(filePath);
+    }
+
+    /**
+     * Loads tasks from the file specified by {@code filePath}.
+     *
+     * @return A list of tasks loaded from the file. Returns an empty list if the file does not exist.
+     * @throws IOException If an I/O error occurs.
+     * @throws XiziException If the data in the file is corrupted or in an unexpected format.
+     */
+    public List<Task> load() throws IOException, XiziException {
```

Comment:
method name could be more descriptive

Reply:
Thanks! I have modified it to loadTasks() !